### ISSUE TITLE
[Accessibility][ScreenReader]: Screen reader does not read correctly when using keyboard arrows to navigate from link back to text.

### ISSUE THREAD

**Issue by rp-dev** [microsoft/BotFramework-WebChat#3116]
## Screenshots

![image](https://user-images.githubusercontent.com/50723361/79808961-2ae60380-8324-11ea-8767-9311461d6e9d.png)

## Version
v4 

## Describe the bug

When user arrives at a link, press "up arrow" with JAWS ON to read the prior statement. JAWS does not read.

## Steps to reproduce

Navigate to link in a message and use the up arrow to go back to text.

## Expected behavior

JAWS should read the text before Terms and Conditions link

## Additional context

Environment:  Windows 10 + Chrome 

[Bug]

**Comment by 1Copenut** [comment]
@rp-dev I noticed this same behavior this morning. I was running some checks using the [axe browser plugin](https://deque.com/axe) and it threw an error `ARIA hidden element must not contain focusable elements`.

After some investigation, it has to do with the `div[aria-hidden="true"]` that wraps each response. The text being read by screen readers is actually a visually hidden `<span>`, so the link is effectively invisible to screen readers unless you are navigating by `TAB` keypress.

VoiceOver for MacOS does not include these links in the Links rotor menu, and I'm guessing JAWS and NVDA do the same.

**Comment by compulim** [comment]
This is fixed in #3287.
